py/test/selenium/webdriver/firefox/firefox_sizing_tests.py

```diff
@@ -30,33 +30,26 @@ def is_running_wayland():
 
 
 @pytest.mark.skipif(not is_running_wayland(), reason="This test only runs on Linux under Wayland")
-def test_firefox_opens_large_when_running_xwayland(request):  # noqa: F821
-    options = Options()
-    if request.config.getoption("--headless"):
-        options.add_argument("-headless")
+def test_firefox_opens_large_when_running_xwayland(headless):
```

Comment:
having a conditional in these tests isn't a good practice. Can we put the conditional in the fixture (maybe call it firefox_options) which adds headless arg if it applies?

Reply:
I did that in a few tests.. I have no idea why :) I'll move it to a fixture.